<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalConfWindown</class>
 <widget class="QDialog" name="SignalConfWindown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transmitter signal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1" rowspan="2">
         <widget class="QFrame" name="frame_4">
          <property name="minimumSize">
           <size>
            <width>515</width>
            <height>480</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QCustomPlot" name="plotWindow" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>515</width>
            <height>60</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="formula">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>410</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>450</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>w</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="AmpLine"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>p1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="tau2Line"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>tau1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Amp</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="wLine"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="M1Line"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>tau2</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>p2</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="p1Line"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>M2</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="p2Line"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="M2Line"/>
           </item>
           <item row="8" column="0" colspan="4">
            <widget class="QFrame" name="frame_6">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>80</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QPushButton" name="saveButton">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="defaultButton">
                <property name="text">
                 <string>Default</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Diagram</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>M1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="tau1Line"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
